pkg/chunkenc/unordered.go

```diff
@@ -88,14 +90,14 @@ func (hb *unorderedHeadBlock) UncompressedSize() int {
 	return hb.size
 }
 
-func (hb *unorderedHeadBlock) Reset() {
-	x := newUnorderedHeadBlock(hb.format)
+func (hb *unorderedHeadBlock) Reset(symbolizer *symbolizer) {
+	x := newUnorderedHeadBlock(hb.format, symbolizer)
 	*hb = *x
 }
 
 type nsEntry struct {
-	line             string
-	nonIndexedLabels labels.Labels
+	line    string
+	symbols symbols
```

Comment:
I think this should be renamed to `nonIndexedLabelsSymbols` to account for what the symbols are used for